src/components/program/ProgramDataBar.jsx

```diff
@@ -0,0 +1,54 @@
+import React, { useContext, useState } from 'react';
+import { Container } from '@edx/paragon';
+import classNames from 'classnames';
+import { Link } from 'react-scroll';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faChevronCircleDown } from '@fortawesome/free-solid-svg-icons';
+import { ProgramContext } from './ProgramContextProvider';
+import ProgramDataBarDetails from './ProgramDataBarDetails';
+
+const ProgramDataBar = () => {
+  const [stuck, setStuck] = useState(false);
+  const {
+    program: { authoringOrganizations: owners, isProgramEligibleForOneClickPurchase },
+  } = useContext(ProgramContext);
+  return (
+    <div
+      className={classNames('data-bar', 'shadow', { stuck })}
+    >
+      <div className={classNames('data-bar-content', `partner-count-${owners.length}`)}>
+        <Container size="lg">
+          <div className="row">
+            <div className="partner-image-wrapper">
+              {owners.map(({ logoImageUrl: imgUrl, name, key }) => (
+                <div className="partner" key={key}>
+                  <img src={imgUrl} alt={name} className="partner-logo" />
+                </div>
+              ))}
+            </div>
+            <ProgramDataBarDetails
+              handleStick={() => setStuck(true)}
+              handleRelease={() => setStuck(false)}
+            />
+            {
+              isProgramEligibleForOneClickPurchase && (
```

Comment:
what would the flow look like when `isProgramEligibleForOneClickPurchase` is false?

Reply:
we wont show them the i am interested button, this is what they have done on b2c side, we doing the same is appropriate?